nat-lab/tests/telio_test.py

```diff
@@ -328,43 +177,14 @@ async def test_explicit_new_derp_state(self) -> None:
         )
 
         await testing.wait_short(
-            events.wait_for_derp_state("1.1.1.1", [State.Connected]),
+            events.wait_for_state_derp("1.1.1.1", [State.Connected])
         )
 
-        # Connection already achieved, should return immediately
+        # It should pass again
         await testing.wait_short(
-            events.wait_for_derp_state("1.1.1.1", [State.Connected]),
+            events.wait_for_state_derp("1.1.1.1", [State.Connected])
         )
 
-        # This should timeout even though derp connection state is connected, since no new event is generated
-        with pytest.raises(asyncio.TimeoutError):
-            await testing.wait_short(
-                events.wait_for_new_derp_state("1.1.1.1", [State.Connected])
-            )
-
-        # Start waiting for new event before it is being generated
-        async with run_async_context(
```

Comment:
why remove this?

Reply:
Removed this test initially because it didn't make sense for states, nevertheless added new tests